sdk/data/azappconfig/internal/generated/azureappconfiguration_client.go

```diff
@@ -766,6 +766,29 @@ func (client *AzureAppConfigurationClient) getLabelsHandleResponse(resp *http.Re
 	return result, nil
 }
 
+// getNextPageCreateRequest creates the GetNextPage request.
+func (client *AzureAppConfigurationClient) getNextPageCreateRequest(ctx context.Context, nextLink string) (*policy.Request, error) {
+	urlPath := nextLink
+	req, err := runtime.NewRequest(ctx, http.MethodGet, runtime.JoinPaths(client.endpoint, urlPath))
+	if err != nil {
+		return nil, err
+	}
+	req.Raw().Header["Accept"] = []string{"application/json"}
+	return req, nil
+}
+
+// getNextPageHandleResponse handles the GetNextPage response.
+func (client *AzureAppConfigurationClient) getNextPageHandleResponse(resp *http.Response) (AzureAppConfigurationClientGetNextPageResponse, error) {
```

Comment:
This, and the response envelope type, are vestigial.  I've opened https://github.com/Azure/autorest.go/issues/895 to track cleaning this up.